### PR TITLE
Correct some issues caused by an older PR

### PR DESCRIPTION
**Pull request by 4mmonium**
Corrects some issues present in the following files:

- CreateCam.md
- AddBlipForEntity.md
- AddBlipForRadius.md
- NetworkGetNetworkIdFromEntity.md
- CreateObject.md

This should also address some issues with non-working js examples.

### CODE REVIEW

File: NETWORK/NetworkGetNetworkIdFromEntity.md [hunk 1]
```diff
@@ -27,11 +28,12 @@ local netId = NetworkGetNetworkIdFromEntity(obj)
 ```
 
 ```js  
-const obj = CreateObject(GetHashKey("prop_weed_01"), vector3(0.0, 0.0, 0.0), true)
-const netId = NetworkGetNetworkIdFromEntity(obj)
+let [x, y, z] = GetEntityCoords(GetPlayerPed(-1));
```
Comment (AvarianKnight) [inline NETWORK/NetworkGetNetworkIdFromEntity.md:31]: It's preached to use `PlayerPedId()` over `GetPlayerPed(-1)`

File: OBJECT/CreateObject.md [hunk 2]
```diff
@@ -41,19 +41,21 @@ local obj = CreateObject(modelHash, vector3(0.0, 0.0, 0.0), true)
...
-        RequestModel(modelHash)
+        RequestModel(modelHash);
         do {
             new Promise(resolve => setTimeout(resolve, 100));
         } while (!HasModelLoaded(modelHash));
     }
 
     // At this moment the model its loaded, so now we can create the object
-    const obj = CreateObject(modelHash, vector3(0.0, 0.0, 0.0), true)
+    let [x, y, z] = GetEntityCoords(GetPlayerPed(-1));
```
Comment (AvarianKnight) [inline OBJECT/CreateObject.md:56]: Same here

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

File: CAM/CreateCam.md [hunk 3]
```diff
@@ -7,23 +7,23 @@ ns: CAM
 // 0xC3981DCE61D9E13F 0xE9BF2A7D
 Cam CREATE_CAM(char* camName, BOOL active);
 ```
-Create a camera with the specified cam name, You can use `SET_CAM_` natives to manipulate the camera.
+Creates a camera with the specified cam name, You can use `SET_CAM_` natives to manipulate the camera.  
+Make sure to call [RENDER_SCRIPT_CAMS](#_0x07E5B515DB0636FC) once the camera is created, or this won't have any visible effect.
+
+**Camera names:**  
+- DEFAULT_SCRIPTED_CAMERA
```
Comment (blattersturm) [inline CAM/CreateCam.md:14]: Technically, there's more in `cameras.#mt`. I'm not sure if any have obvious names, but it'd be useful to check for any script cameras in there.

File: CAM/CreateCam.md [hunk 4]
```diff
@@ -7,23 +7,23 @@ ns: CAM
 // 0xC3981DCE61D9E13F 0xE9BF2A7D
 Cam CREATE_CAM(char* camName, BOOL active);
 ```
-Create a camera with the specified cam name, You can use `SET_CAM_` natives to manipulate the camera.
+Creates a camera with the specified cam name, You can use `SET_CAM_` natives to manipulate the camera.  
+Make sure to call [RENDER_SCRIPT_CAMS](#_0x07E5B515DB0636FC) once the camera is created, or this won't have any visible effect.
+
+**Camera names:**
```
Comment (blattersturm) [inline CAM/CreateCam.md:13]: Using `###`-style headings should work fine.

File: HUD/AddBlipForEntity.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ ns: HUD
 // 0x5CDE92C702A8FCE7 0x30822554
 Blip ADD_BLIP_FOR_ENTITY(Entity entity);
 ```
-Create a blip that for default is red (enemy), you can use the [SET_BLIP_AS_FRIENDLY](#_0xC6F43D0E) to make it blue (friend)
+Create a blip that for default is red (enemy), you can use [SET_BLIP_AS_FRIENDLY](#_0xC6F43D0E) to make it blue (friend)
```
Comment (blattersturm) [inline HUD/AddBlipForEntity.md:10]: for default -> by default

not sure if this applies to non-peds either.

File: NETWORK/NetworkGetNetworkIdFromEntity.md [hunk 6]
```diff
@@ -9,11 +9,12 @@ int NETWORK_GET_NETWORK_ID_FROM_ENTITY(Entity entity);
 ```
 Get the network id of the given entity
 
-An entity network id is representing a given entity for other clients, is like a handle that every client recognize, can be reconverted to a client entity handle. 
-These can and will be reused, the network id will not change.
+An entity network id represents a given entity for other clients, it's a handle shared between clients, can be reconverted to a client entity handle.  
+These (entity network id's) can and will be reused, the network id will not change.
```
Comment (blattersturm) [inline NETWORK/NetworkGetNetworkIdFromEntity.md:13]: id's -> IDs

**Review by blattersturm — CHANGES_REQUESTED** [review]

File: CAM/CreateCam.md (hunk 3 above)
Comment (4mmonium) [reply CAM/CreateCam.md:14]: Will look into that, thanks

File: NETWORK/NetworkGetNetworkIdFromEntity.md (hunk 1 above)
Comment (4mmonium) [reply NETWORK/NetworkGetNetworkIdFromEntity.md:31]: Yeah, looks way cleaner that way 👍

**Comment by 4mmonium** [comment]
Updated PR with the following [commit](https://github.com/citizenfx/natives/pull/801/commits/34f6327d750574cde9deba844b96a2984d5f06dc), found these camera names, but I didn't see any effect upon usage of `CreateCam`, hence why I didn't add them to the native. Would hate to see people try stuff that doesn't work accordingly.

- DEFAULT_FREE_CAMERA
- SYNCED_SCENE_DIRECTOR
- SCRIPT_DIRECTOR
- SWITCH_DIRECTOR
- CUTSCENE_DIRECTOR
- REPLAY_DIRECTOR
- DEBUG_DIRECTOR
- CINEMATIC_ON_FOOT_IDLE_CONTEXT
- CINEMATIC_IN_VEHICLE_FIRST_PERSON_CONTEXT
- CINEMATIC_STUNT_JUMP_CONTEXT
- GAMEPLAY_DIRECTOR
- CINEMATIC_ON_FOOT_MELEE_CONTEXT
- CINEMATIC_PARACHUTE_CONTEXT
- CINEMATIC_IN_TRAIN_CONTEXT
- CINEMATIC_IN_TRAIN_AT_STATION_CONTEXT
- CINEMATIC_IN_VEHICLE_CONTEXT
- CINEMATIC_IN_VEHICLE_WANTED_CONTEXT
- CINEMATIC_IN_VEHICLE_OVERRIDDEN_FIRST_PERSON_CONTEXT
- CINEMATIC_BUSTED_CONTEXT
- CINEMATIC_SCRIPTED_RACE_CHECK_POINT_CONTEXT
- CINEMATIC_SCRIPT_CONTEXT
- CINEMATIC_SPECTATOR_NEWS_CHANNEL_CONTEXT
- CINEMATIC_DIRECTOR
- MARKETING_DIRECTOR
- A_TO_B_CAMERA

File: HUD/AddBlipForEntity.md (hunk 5 above)
Comment (4mmonium) [reply HUD/AddBlipForEntity.md]: Tested and I could make objects, as well as vehicles 'enemies'.

**Pull request merged**